follow.go

```diff
@@ -121,3 +122,86 @@ func following(ctx context.Context, c *client, userID string, opt ...*FollowOpti
 
 	return &f, nil
 }
+
+// tuid = target_user_id
+func postFollowing(ctx context.Context, c *client, id string, tuid string) (*PostFollowingResponse, error) {
+	if id == "" {
+		return nil, errors.New("post following by id: id parameter is required")
```

Comment:
```suggestion
		return nil, errors.New("post following by id: id parameter is required")
```

I think ID is not clearly.